variables.tf

```diff
@@ -72,11 +72,16 @@ variable "public_network_acl_id" {
 }
 
 variable "private_network_acl_id" {
-  default     = ""
   description = "Network ACL ID that will be added to private subnets. If empty, a new ACL will be created"
+  default     = ""
 }
 
 variable "nat_gateway_enabled" {
   description = "Flag to enable/disable NAT gateways for private subnets"
-  default     = "true"
+  default     = true
```

Comment:
@osulli 
1) `nat_gateway_enabled` is tested for the string "true" (not boolean) here   https://github.com/cloudposse/terraform-aws-dynamic-subnets/blob/master/nat.tf#L11, so either `default     = "true"` or it needs to be updated everywhere else
2) we should not make other changes not related to this PR - difficult to track what was changed and why

Reply:
@osulli please make `default     = "true"` again (back to `string` type since here https://github.com/cloudposse/terraform-aws-dynamic-subnets/blob/master/nat.tf#L11 it's checked as a string

Reply:
Done